src/runtime/Object.cpp

```diff
@@ -779,20 +784,17 @@ ValueVector Object::ownPropertyKeys(ExecutionState& state)
 // https://www.ecma-international.org/ecma-262/6.0/#sec-ordinary-object-internal-methods-and-internal-slots-get-p-receiver
 ObjectGetResult Object::get(ExecutionState& state, const ObjectPropertyName& propertyName)
 {
-    Object* temp = this;
-    auto desc = this->getOwnProperty(state, propertyName);
-    while (true) {
-        if (!desc.hasValue()) {
```

Comment:
Why did you fix this `Object::get` method? Is there any issue related to the feature?

Reply:
This change is a reverts the function to it's original state. The reason is that in ES5.1 we could simply iterate through the prototype chain with a single while loop, since there was only one [[Get]] internal method. That's why the recursive algorithm of the standard could be changed into the iterative one. However, in ES6 there are proxy objects. When you call a `obj->get` for a simple object who has a proxy object in  its prototype chain, you must invoke the ProxyObject::get during the lookup, as the standard says. Currently we use iterative implementation, which invokes the ProxyObject::getOwnProperty during the lookup. I hope you see my point.

Reply:
I see. It looks inevitable. Maybe we could optimize `Object::get` later if possible.

Reply:
I think I found a solution for not using recursion so probably i can resolve this problem.